maintainers.yaml

```diff
@@ -0,0 +1,9 @@
+extra:
+   recipe-maintainers:
```

Comment:
Are the GitHub usernames case sensitive?

Reply:
I think so too.

Reply:
I'll change that :-)